### PR TITLE
Modified EVM Resources section

### PR DESCRIPTION
**Pull request by zaryab2000**
This PR provides an enhanced structure in the EVM learning resources section and mainly include 3 important changes:

- **Enhanced structure for EVM learning resources:** Instead of dropping all resources at once, it categorizes resources based on level of expertise so it's easier for beginners to navigate.
- **Inclusion of some more EVM resources**
- **Inclusion of some well-known EVM CTF Puzzles**

### CODE REVIEW

File: docs/wiki/EL/evm.md [hunk 1]
```diff
@@ -308,27 +308,44 @@ While Ethereum protocol undergoes many changes in each upgrade, changes in EVM a
 
 ### EVM
 
+The resources below has been effectively arranged based on your level of expertise.
+
+#### Level Zero: Beginner
```
Comment (taxmeifyoucan) [inline docs/wiki/EL/evm.md:313]: I like the idea of ordering these from beginner to advanced but I wouldn't necessarily call it levels
```suggestion
#### Beginner
```

Comment (zaryab2000) [reply docs/wiki/EL/evm.md:313]: Sure makes sense.

lemme know if you have any suggestions, I will push changes accordingly.

Additionaly working on improving the WIKI section contents too , will create more PRs soon.

Comment (taxmeifyoucan) [reply docs/wiki/EL/evm.md:313]: Rest of your changes look good, I think  new resources are great!. I would just rename some of these titles and we can get it merged

Comment (zaryab2000) [reply docs/wiki/EL/evm.md:313]: Sure, that sounds good.

Feel free to change titles as you see fit. Just lemme know if you need my help in changing anything. Thanks @taxmeifyoucan

Comment (zaryab2000) [reply docs/wiki/EL/evm.md]: I just pushed a commit:

-  to fix titles, 
- removed "level" as rightly pointed out
- fixed some texts and punctuation as well.

Lemme know if any more changes are required in this branch. @taxmeifyoucan . Thanks.

Comment (taxmeifyoucan) [reply docs/wiki/EL/evm.md]: thanks, all looks good!

**Pull request merged**